pkg/appgw/backendhttpsettings.go

```diff
@@ -182,5 +187,30 @@ func (builder *appGwConfigBuilder) generateHTTPSettings(backendID backendIdentif
 	if err == nil {
 		httpSettings.Path = to.StringPtr(pathPrefix)
 	}
+
+	isConnDrain, err := annotations.IsConnectionDraining(backendID.Ingress)
```

Comment:
You could (if you wanted to) inline this to a) limit the scope of the 2 variables and b) play some code golf:
```go
if isConnDrain, err := annotations.IsConnectionDraining(backendID.Ingress); if err == nil && isConnDrain {
```